### ISSUE TITLE
Update may change file's encoding

### ISSUE THREAD

**Issue by NeeXxx** [microsoft/vscode#98318]
- VSCode Version: 1.45.1
- OS Version: macOS Sierra 10.12.6

Steps to Reproduce:

1. I click 'reload to update' button
2. the encoding of one of my file change from 'utf8' to 'ascii'. (I used the `file` command to see its encoding, I believe this means that the origin information is forever lost and cannot be restore by choosing different encoding.)
3. This happens twice. Last time I thought it was caused by certain extension, and I deleted a lot extensions.
4. I'm sorry I cannot clearly reproduce the problem, and this does not happen every time I update, but it's really annoying. Any help will be appreciated.

Does this issue occur when all extensions are disabled?: Not sure

**Comment by bpasero** [comment]
If you can distill reliable steps to reproduce. Do you maybe have language specific settings for encoding?

**Comment by NeeXxx** [comment]
`Auto Guess Encoding` is True.

**Comment by bpasero** [comment]
Does it reproduce if you just reload VSCode?

**Comment by NeeXxx** [comment]
Reload VSCode can’t reproduce it.

Here’s my activities:

Two of my files suffer the issue. I noticed File A then File B.

File A:
1. I opened the file with VSCode, native characters turned into ‘?’.
2. Then I opened it with PyCharm, native characters displayed correctly, but after a few seconds it turned into ‘?’ too.
3. I opened it with sublime, native characters display normally.
4. I reloaded VSCode, native characters display normally.

File B:
It was found after I reloaded VSCode. And it just kept display ‘?’ with every editors.
